charts/nexus-iq/templates/persistentVolumeClaim.yaml

```diff
@@ -1,28 +1,30 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: {{ template "iqserver.fullname" . }}-data
   labels:
-    {{- include "iqserver.labels" . | nindent 4 }}
+{{ include "iqserver.labels" . | indent 4 }}
     {{- if .Values.iq.extraLabels }}
       {{- with .Values.iq.extraLabels }}
         {{ toYaml . | indent 4 }}
       {{- end }}
     {{- end }}
-  {{- if .Values.persistence.annotations }}
-annotations:
-  {{ toYaml .Values.persistence.annotations | indent 2 }}
-  {{- end }}
+{{- if .Values.persistence.annotations }}
+  annotations:
+{{ toYaml .Values.persistence.annotations | indent 4 }}
```

Comment:
is it possible to have the `toYaml` call to be indented so that it looks like it's nested under `annotations`?